app/src/main/java/com/kickstarter/features/search/ui/SearchAndFilterActivity.kt

```diff
@@ -59,6 +60,8 @@ class SearchAndFilterActivity : ComponentActivity() {
             filterMenuViewModelFactory = FilterMenuViewModel.Factory(env)
             filterMenuViewModel.getRootCategories()
 
+            val phase2ff = env.featureFlagClient()?.getBoolean(FlagKey.ANDROID_SEARCH_FILTER) ?: false
```

Comment:
If don't wanna qa by turning off Remote Config feature flag, you can always change values here:
`val phase2ff = false` -> in this way you can simulate remote config ff off